LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -167,6 +160,33 @@ extension TherapySettingsView {
         }
     }
     
+    private var suspendThreshold: some View {
+        section(for: .suspendThreshold) {
+            Group {
+                if self.glucoseUnit != nil {
+                    HStack {
+                        Spacer()
+                        GuardrailConstrainedQuantityView(
+                            value: self.therapySettings.suspendThreshold?.quantity,
+                            unit: self.glucoseUnit!,
+                            guardrail: .suspendThreshold,
+                            isEditing: false,
+                            // Workaround for strange animation behavior on appearance
+                            forceDisableAnimations: true
+                        )
+                    }
+                }
+            }
+        }
+    }
+    
+    private func section<Content>(for therapySetting: TherapySetting, content: @escaping () -> Content) -> some View where Content: View {
```

Comment:
If you annotate the `content` parameter with `@ViewBuilder`, you can drop the use of Group that you have in instances of the closure passed to this method